daml-lf/spec/daml-lf-1.rst

```diff
@@ -1882,6 +1956,38 @@ need to be evaluated further. ::
    ——————————————————————————————————————————————————— ValScenario
      ⊢ᵥ  s
 
+   ——————————————————————————————————————————————————— ValMathCtxUnlimited
+     ⊢ᵥ  'MATHCTX_UNLIMITED'
+
+     ⊢ᵥ  r
+     ⊢ᵥ  p
+   ——————————————————————————————————————————————————— ValMathCtxBounded
+     ⊢ᵥ  'MATHCTX_BOUNDED' r p
+
+   ——————————————————————————————————————————————————— ValRoundingModeCeiling
+     ⊢ᵥ  'ROUNDING_CEILING'
+
+   ——————————————————————————————————————————————————— ValRoundingModeFloor
+     ⊢ᵥ  'ROUNDING_FLOOR'
+
+   ——————————————————————————————————————————————————— ValRoundingModeDown
+     ⊢ᵥ  'ROUNDING_DOWN'
+
+   ——————————————————————————————————————————————————— ValRoundingModeUp
+     ⊢ᵥ  'ROUNDING_UP'
+
+   ——————————————————————————————————————————————————— ValRoundingModeHalf_Down
+     ⊢ᵥ  'ROUNDING_HALF_DOWN'
+
+   ——————————————————————————————————————————————————— ValRoundingModeHalf_Even
+     ⊢ᵥ  'ROUNDING_HALF_EVEN'
+
+   ——————————————————————————————————————————————————— ValRoundingModeHalf_Up
+     ⊢ᵥ  'ROUNDING_HALF_UP'
+
+   ——————————————————————————————————————————————————— ValRoundingModeHalf_Unnecessary
+     ⊢ᵥ  'ROUNDING_HALF_UNNECESSARY'
```

Comment:
The value rules are good. They don't need to be changed if you make these built-in functions. :-)